tests/topotests/bgp_evpn_rt5/test_bgp_evpn.py

```diff
@@ -208,6 +215,52 @@ def test_protocols_convergence():
     else:
         logger.info("Check Ping IPv4 from R1(r1-vrf-101) to R2(192.168.101.41) OK")
 
+    logger.info(
+        "Check Ping IPv6 from  R1(r1-vrf-101) to R2(r2-vrf-101 = fd00::2)"
+    )
+    output = pingrouter.run("ip netns exec r1-vrf-101 ping fd00::2 -f -c 1000")
+    logger.info(output)
+    if "1000 packets transmitted, 1000 received" not in output:
+        assertmsg = (
+            "expected ping IPv4 from R1(r1-vrf-101) to R2(fd00::2) should be ok"
+        )
+        assert 0, assertmsg
+    else:
+        logger.info("Check Ping IPv4 from R1(r1-vrf-101) to R2(192.168.101.41) OK")
+
+    config_no_ipv6 = {
+        "r2": {
+            "raw_config": [
+                "router bgp 65000 vrf r2-vrf-101",
+                "address-family ipv6 unicast",
+                "no network fd00::2/128"
+            ]
+        }
+    }
+
+    logger.info("==== Remove IPv6 network on R2")
+    result = apply_raw_config(tgen, config_no_ipv6)
+    assert result is True, "Failed to remove IPv6 network on R2, Error: {} ".format(result)
+    topotest.sleep(4, "waiting 4 seconds for bgp convergence")
```

Comment:
Nah, sleep is not an acceptable option, can we use something here with retries?

Reply:
Replaced `sleep` with `verify_bgp_rib`.